common4j/src/main/com/microsoft/identity/common/java/authorities/Authority.java

```diff
@@ -78,9 +78,21 @@ public void setSlice(AzureActiveDirectorySlice slice) {
         mSlice = slice;
     }
 
-    public abstract Uri getAuthorityUri();
+    public URI getAuthorityUri() {
+        try {
+            return new URI(mAuthorityUrl);
```

Comment:
to save people like me from wondering why we're converting a URL to a URI and back again, maybe rename this variable to mAuthorityUrlString?